packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -1627,7 +1627,12 @@ export default function ToolpadApp({ rootRef, basename, state }: ToolpadAppProps
     (window as any).toggleDevtools = () => toggleDevtools();
   }, [toggleDevtools]);
 
-  const authContext = useAuth({ dom, basename, isRenderedInCanvas: IS_RENDERED_IN_CANVAS });
+  const authContext = useAuth({
```

Comment:
No need to pass this around anymore, can get it from context, see https://github.com/mui/mui-toolpad/pull/3170/